packages/web-runtime/src/container/application/index.ts

```diff
@@ -122,7 +122,7 @@ export const buildApplication = async ({
     if (!isObject(applicationScript.appInfo) && !applicationScript.setup) {
       throw new RuntimeError('next applications not implemented yet, stay tuned')
```

Comment:
We should get rid of Next Applications as a concept... Classic and Next Applications are somewhat converging anyhow